.gitignore

```diff
@@ -34,7 +34,6 @@ build
 node_modules
 
 lib
-dist
```

Comment:
The dist should be ignored. The dist folder will be created on the release script for publishing but should not be part of the repo.

Reply:
So in this case, do we leave the package.json pointing to `dist/src/index.d.ts` even though it's no longer included in my PR?

Reply:
Yes, it will be created in the release and what is shipped to npm will include the dist folder